src/components/Details.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+const Details = ({ details, setDetails }) => {
+  let numOfPurch = details.purchaseDates.length;
+  const lastPurch = numOfPurch
+    ? Math.max(...details.purchaseDates)
+    : details.addedDate;
+  let futurePurch = lastPurch + details.frequency;
```

Comment:
It's nice to be consistent with `let` and `const`, when possible. In this case, they could be all `let` or all `const`, as nothing is being assigned, right?